src/Symfony/Component/Config/Definition/Builder/NodeFinder.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Definition\Builder;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class NodeFinder
+{
+    public function find(string $nodePath, NodeDefinition $rootNode)
```

Comment:
Please add a description of the format of „$nodePath“

Reply:
Done